services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import Ticket, Order, User, MovieSession
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(tickets: list[Ticket],
+                 username: str,
+                 date: str = None) -> QuerySet:
+    user = User.objects.get(username=username)
+
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = datetime.strptime(date, "%Y-%m-%d %H:%M")
+        order.save(update_fields=["created_at"])
+
+    for ticket_data in tickets:
+        movie_session = (MovieSession.objects
+                         .get(id=ticket_data["movie_session"]))
```

Comment:
The `get` method is used here, which can raise a `DoesNotExist` exception if no `MovieSession` object with the provided id is found. You should handle this potential exception.